### DISCUSSION TITLE
Check Prior Messages for Duplicate ICD-10 or CPT Codes

### DISCUSSION THREAD

**Discussion by Che-jj** [nextgenhealthcare/connect#5511] [Q&A] [answered]
Mirth 3.8.1 running on Ubuntu 20.04.1
The LIS I work with triggers ORU and DFT messages when a report is signed out. When an amended report or corrected report is signed out, it will retrigger and send out a new ORU with the corrected/amended results and a new DFT with any new CPT/ICD-10 codes and all of the old ones as well. Is there any function I can run in the transformer to check prior messages in a channel? I want to see which CPT/ICD10 codes on the amended / corrected DFT were already on the original DFT message to avoid a duplicate billing issue.

**Comment by pacmano1** [comment, marked as answer]
There are similar posts about checking for "duplicates" here on GH and on the forums.  

Effectively you need to write code for all that.
